src/lib/AlertWindow.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import ModalWindow from "./ModalWindow";
+import Button from "./Button";
+
+const InfoText = styled.div`
+  text-align: center;
+`;
+
+const ButtonRow = styled.div`
+  text-align: center;
+  margin-top: 32px;
+  
+  button {
+    margin: 0 16px;
+  }
+`;
+
+const AlertWindow = (props) => (
+  <ModalWindow {...props}>
```

Comment:
When passing extra props to the child, you should omit the component's own props (unless they are really needed), either by `const { info, buttons, ...rest } = props` or with Lodash.omit. 
Then:
`<ModalWindow {...rest}>`

Reply:
Done